Clinical-Genomics/scout
# Truncate long snvs names in general report

fix #1760 
When snv variants have very long display names the general report looks ugly:

![image](https://user-images.githubusercontent.com/28093618/75357517-cbceba00-58b1-11ea-9630-baabf873cf1b.png)


**How to test**:
1. On a local instance of mongodb change display name of this variant : _id: a76ebd54e531bb276638544b187c14fd to `17_7470283_ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC_A_clinical`
1. Make sure to modify also variant reference to `ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC`
1. Modify also simple id to `17_7470283_ACGGGTAAAACCATTGGTATTATATTAGGCCCGAGGAGACTTTCTCTCTCTTCGGGGAGAGGCTTCTCTCTCTTCTCTC_A`

1. On a local installation of scout go to the same variant: http://localhost:5000/cust000/643594/a76ebd54e531bb276638544b187c14fd
 and mark it as causative, then dismiss it with many reasons, including the "inherited from unaffected" tag.
1. Check the difference in the general report using master and this branch

**Expected outcome**:
Report should look fine when using this branch, specifically:
- [ ] The header of the causative variant
- [ ] The columns of the dismissed variants
- [ ] Simulate a clinvar export of the variant and make sure that the clinvar form looks fine.
- [ ] On the variant page the reference allele should be also truncated and on its side there should be an icon that allows you to copy the ref allele to clipboard
- [ ] Same should be possible if the long string is the alternative allele.

Take a screenshot and attach or copy/paste the output.

**Review:**
